packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -621,17 +610,14 @@ export class MgtAgenda extends MgtTemplatedComponent {
   }
 
   private prettyPrintTimeFromDateTime(date: Date) {
-    // If a preferred time zone was sent in the Graph request
-    // times are already set correctly. Do not adjust
-    if (!this.preferredTimezone) {
```

Comment:
It's not necessary. The adjustment being done here is to set the minutes of the date to the value of the current minutes with a timezone offset. 

Since the date string is in UTC, the changes are to display it as it is hence the customers without `preferredTimezone` attributes will still have dates represented in the value returned from the API.

The change does a similar thing. The only difference is that a `new Date` object is created with the current date object time that is in millisecond and the time zone offset subtracted from it (which is also in milliseconds).